content/doc/tutorials/build-a-python-app-with-pyinstaller.adoc

```diff
@@ -62,136 +55,96 @@ Obtain the simple "add" Python application from GitHub, by forking the sample
 
+Now, log in using the `admin` username and `admin` password.
+
+=== Create your Pipeline project in Jenkins
+
+. In Jenkins, select *New Item* under *Dashboard >* at the top left.
+. Enter your new Pipeline project name in *Enter an item name* (e.g. `simple-python-pyinstaller-app`).
+. Scroll down if necessary and select *Pipeline*, then click *OK* at the end of the page.
+. (Optional) Enter a Pipeline *Description*.
+. Select *Pipeline* on the left pane.
+. Select *Definition*, and then choose the *Pipeline script from SCM* option.
+This option instructs Jenkins to obtain your Pipeline from the source control management (SCM), which is your forked Git repository.
+. Choose *Git* from the options in *SCM*.
+. Enter the URL of your repository in *Repositories/Repository URL*. This URL can be found when clicking on the green button *Code* in the main page of your GitHub repo.
+. Hit the *Save* button at the end of the page.
+You're now ready to create a `Jenkinsfile` to check into your locally cloned Git repository.
 
 === Create your initial Pipeline as a Jenkinsfile
 
-You're now ready to create your Pipeline that will automate building your Python
-application with PyInstaller in Jenkins. Your Pipeline will be created as a
-`Jenkinsfile`, which will be committed to your locally cloned Git repository
-(`simple-python-pyinstaller-app`).
-
-This is the foundation of "Pipeline-as-Code", which treats the continuous
-delivery pipeline a part of the application to be versioned and reviewed like
-any other code. Read more about Pipeline and what a Jenkinsfile is in the
-link:/doc/book/pipeline[Pipeline] and link:/doc/book/pipeline/jenkinsfile[Using
-a Jenkinsfile] sections of the User Handbook.
-
-First, create an initial Pipeline with a "Build" stage that executes the first
-part of the entire production process for your application. This "Build" stage
-downloads a Python Docker image and runs it as a Docker container, which in turn
-compiles your simple Python application into byte code.
-
-. Using your favorite text editor or IDE, create and save new text file with the
-  name `Jenkinsfile` at the root of your local `simple-python-pyinstaller-app`
-  Git repository.
-. Copy the following Declarative Pipeline code and paste it into your empty
-  `Jenkinsfile`:
+You're now ready to create your Pipeline that will automate building your Python application with PyInstaller in Jenkins.
+Your Pipeline will be created as a `Jenkinsfile`, which will be committed to your locally cloned Git repository (`simple-python-pyinstaller-app`), and then pushed to GitHub, where Jenkins will be able to find it.
+
+This is the foundation of "Pipeline-as-Code", which treats the continuous delivery pipeline a part of the application to be versioned and reviewed like any other code.
```

Comment:
```suggestion
This is the foundation of "Pipeline-as-Code", which treats the continuous delivery pipeline as part of the application to be versioned and reviewed like any other code.
```
Updating "a" to "as" so that it makes sense in context